Automattic/woocommerce-payments
# Fix fatal issue if enabled currencies db option is set as string.

Fixes #2710 

#### Changes proposed in this Pull Request

* Check that `wcpay_multi_currency_enabled_currencies` option returns an array, and if not, make it an empty array.
* The reason for making an empty array is that we've never set this option as anything other than array, so we wouldn't know how to read any string entered.
* Check that currency cache returned from db is an array before checking for array values.

#### Testing instructions

1. Go into your `wp_options` table and delete the `option_value` for `wcpay_multi_currency_enabled_currencies`.
2. If you don't have this option, create a new row with `wcpay_multi_currency_enabled_currencies` as the `option_value`.
2. View any page on the site, no error should be received.
3. Navigate to WooCommerce > Settings > Multi-Currency, the store currency should show as the only enabled currency.

-------------------
